docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -1473,7 +1473,7 @@ memory in a runtime class loader. To have these classes written out to disk for
 
 [source,shell]
 ----
-./mvnw clean install -Dquarkus.debug.generated-classes-dir=./target/app-generated-classes
+mvn clean install -Dquarkus.debug.generated-classes-dir=./target/app-generated-classes
```

Comment:
What's the rationale behind this change?

Reply:
Hi @geoand thanks for the review. 

This make it possible to C&P the command and launch it from the extension directory where the maven wrapper is not available.

Reply:
Makes sense. I looked at the what is used in other parts of the guide and this is consistent. 👍